tools/roslaunch/test/xml/test-arg-all.xml

```diff
@@ -0,0 +1,48 @@
+<launch>
+
+  <arg name="required" />
+  <arg name="optional" default="not_set" />
```

Comment:
I believe this would affect the test for `/notall_optional/include_test/p2_test`